### PR TITLE
MUMUP-2056 : Time and absence portlet

### PR DESCRIPTION
**Pull request by timlevett**
Based on work done here: https://gist.github.com/timlevett/61951dc9f51e38ba0dfd

#### Loaded Good State Hidden
screenshot 2015-08-18 at 1 35 09 pm 

#### Loaded Good State Show
screenshot 2015-08-18 at 1 38 18 pm

#### Loaded Good State Entry Hover
_I feel like this is really important for people with multiple jobs on campus_
screenshot 2015-08-18 at 1 35 58 pm 

#### Loading
screenshot 2015-08-18 at 1 37 44 pm

#### Error/Empty
screenshot 2015-08-18 at 1 37 32 pm

### CODE REVIEW

File: angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc [hunk 1]
```diff
@@ -169,9 +169,9 @@
...
...lat'><span ng-if='!show'>Show</span><span ng-if='show'>Hide</span> balances</button></div> <div class='col-xs-2'>&nbsp;</div></div></div></div> <div><loading-gif data-object='content' data-empty='isEmpty'></loading-gif><div ng-if='isEmpty' style='padding: 10px; font-size: 14px;'><i class='fa fa-exclamation-triangle fa-3x pull-left' style='color: #b70101;'></i><span style='color: #898989;'>We had a problem fetching your balances.</span></div></div> <ul class='widget-list'> <li ng-repeat=\"item in filteredArray(content.report,'entitlement',['Sick','Vacation Ava','Sabbatical', 'Personal']) | orderBy:'entitlement' | limitTo:5\" ng-init='details=false' ng-mouseenter='details=true' ng-mouseleave='details=false'> <p class='bold'>{{item.entitlement}}<span class='right' ng-if='show'>{{item.balance}}</span><span class='right' style='font-style: italic; font-weight: 100;' ng-if='!show'>hidden</span> </p><p ng-if='details'>{{item.job.title}}</p> </li> <li ng-if='content.report.length > 5' style='font-style: italic; text-align: center;'>Showing 5 out of {{ content.report.length}} </li> </ul></div><a class='btn btn-default launch-app-button ng-scope' href='{{portlet.url}}'>See all balances</a>",
```
Comment (timlevett) [inline angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc]: Pretty view of template can be found [here](https://gist.github.com/timlevett/61951dc9f51e38ba0dfd)

**Comment by timlevett** [comment]
Removed hover state to show job title based on feedback from @keirserrie

**Comment by vertein** [comment]
What are the thoughts on hiding the show balances button while loading as well?  :+1:

**Comment by timlevett** [comment]
Yeah, makes sense @vertein. I'll fix that right quick

**Comment by jhanstra** [comment]
:+1:

**Pull request merged**